person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+
+  attr_reader :id
```

Comment:
[OPTIONAL]

- For best practices, always use `attr_accessor` and `attr_reader` on the top, just below the class.

Reply:
Thank you for the suggestion, @Dipeshtwis